spec/models/scanned_resource_spec.rb

```diff
@@ -174,4 +174,33 @@
       expect(subject.valid?).to eq false
     end
   end
+
+  describe "#check_completion" do
+    it "completes record when state changes to 'complete'" do
+      allow(subject).to receive("state_changed?").and_return true
+      subject.state = 'complete'
+      expect(subject).to receive(:complete_record)
+      expect { subject.check_completion }.to change { ActionMailer::Base.deliveries.count }.by(1)
```

Comment:
This test fails for me -- but the corresponding action works when I use the app locally (the email is dumped to the Rails log instead of sent).